Jenkinsfile

```diff
@@ -55,6 +55,9 @@ spec:
            description: 'URL required for downloading GlassFish Full/Web profile bundle' )
     choice(name: 'PROFILE', choices: 'FULL\nWEB', 
            description: 'Profile to be used for running CTS either web/full' )
+	string(name: 'TCK_BUNDLE_FILE_NAME', 
+           defaultValue: 'cdi-tck-glassfish-porting-2.0_latest.zip', 
```

Comment:
Shouldn't that be "2.0.0"?

Reply:
Thanks for pointing out.I missed those files.have made a new commit which takes care of these changes